ui/lib/sync/addon/components/secrets/page/destinations/create-and-edit.hbs

```diff
@@ -12,18 +12,26 @@
 
   {{#each @destination.formFieldGroups as |fieldGroup|}}
     {{#each-in fieldGroup as |group fields|}}
-      {{#if (and (eq group "Credentials") (not @destination.isNew))}}
+      {{! h2 titles Credentials and Configuration for FormFieldGroups to be displayed in both create and edit }}
+      {{#if (eq group "Credentials")}}
         <hr class="has-top-margin-xl has-bottom-margin-l has-background-gray-200" />
         <Hds::Text::Display @tag="h2" @size="400" @weight="bold">Credentials</Hds::Text::Display>
         <Hds::Text::Body @tag="p" @size="100" @color="faint" class="has-bottom-margin-m">
-          Connection credentials are sensitive information and the value cannot be read. Enable the input to update.
+          Connection credentials are sensitive information
+          {{#if @destination.isNew}} used to authenticate with the destination. {{else}}
+            and the value cannot be read. Enable the input to update.
+          {{/if}}
```

Comment:
Can we add an assertion so this subtext is correct for new/edit forms?

Reply:
Done and dusted. Note the maskedInput vs wrapped maskedInputs by the EnableInput component on create vs edit are covered in the create-and-edit test. They're a little hard to see. I added on for create in this [PR](https://github.com/hashicorp/vault/pull/27348/files#diff-764822d55a50d3a34ccd084f0258acc870d20e566c17448441215927aace7981). And the edit masked check is done by calling enableInput on the test [here](https://github.com/hashicorp/vault/blob/main/ui/tests/integration/components/sync/secrets/page/destinations/create-and-edit-test.js#L377). If those values were not wrapped by the correct component, you couldn't find this value.